x/staking/types/expected_keepers.go

```diff
@@ -115,3 +116,7 @@ type StakingHooksWrapper struct{ StakingHooks }
 
 // IsOnePerModuleType implements the depinject.OnePerModuleType interface.
 func (StakingHooksWrapper) IsOnePerModuleType() {}
+
+type ConsensusKeeper interface {
```

Comment:
This interface was introduced but I can't find any usages. It seems it could be an alternative to using the RouterService to get Conensensus params.

It should be removed right?